lib/rubygems/commands/rebuild_command.rb

```diff
@@ -0,0 +1,264 @@
+# frozen_string_literal: true
+
+require "date"
+require "digest"
+require "fileutils"
+require "tmpdir"
+require_relative "../gemspec_helpers"
+require_relative "../package"
+
+class Gem::Commands::RebuildCommand < Gem::Command
+  include Gem::GemspecHelpers
+
+  DATE_FORMAT = "%Y-%m-%d %H:%M:%S.%N Z"
```

Comment:
What is this constant for?
Seems unused.

Reply:
@nobu good catch, it is unused. I opened #7546 to remove it. :+1: